### PR TITLE
Fix #351: Move probe-dictionary into TMO

### LINKED ISSUE #351: Move data explorer / probe dictionary into TMO project
We decided to move the [data explorer](https://github.com/georgf/fx-data-explorer) into the TMO project.

### PR DESCRIPTION
**Pull request by georgf**
Addressing #351, we want to copy in the probe-dictionary for now.
In the future we want to host from the mozilla/probe-dictionary repo and pull it in as a submodule, but that will require some research first.

### CODE REVIEW

**Comment by chutten** [comment]
@georgf Looks like this needs a rebase to be considered? There are a lot of commits in here that seem unrelated.

**Comment by georgf** [comment]
@chutten This is rebased and the commits won't be visible after squashing.
Best just look at "files changed".

File: probe-dictionary/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## probe-dictionary
+A tool that makes the Telemetry probe data in Firefox more discoverable and searchable.
+
+This front-end allows answering questions like 
+* *"do we have any probes in Firefox 55 that tell us about [tab usage](http://georgf.github.io/fx-data-explorer/?search=tab&searchtype=in_name&optout=true&channel=release&constraint=is_in&version=55)?"*
```
Comment (chutten) [inline probe-dictionary/README.md]: This URL'll need an update

**Review by chutten — APPROVED** [review]
One small thing which you can chose to leave in if you want a good first bug : )

**Pull request merged**